kubejs/server_scripts/enigmatica/kubejs/constants/stonecuttables.js

```diff
@@ -2455,13 +2457,19 @@ chiselPlankVariants = [
     'braid',
     'log_cabin'
 ];
-chiselPlankTypes.forEach((chiselPlankType) => {
-    let stones = [`minecraft:${chiselPlankType}_planks`];
-    chiselPlankVariants.forEach((chiselPlankVariant) => {
-        stones.push(`chisel:planks/${chiselPlankType}/${chiselPlankVariant}`);
-    });
+plankTypes.forEach((plankType) => {
+	const numberChippedVariants = 18;
```

Comment:
Please put this outside the loop :)

Reply:
Oops yeah that probably shouldn't go there